stabilizer/src/main/java/com/hazelcast/stabilizer/provisioner/Provisioner.java

```diff
@@ -343,6 +345,40 @@ public void download(String dir) {
         echoImportant("Finished Downloading Artifacts of %s machines", addresses.size());
     }
 
+    public List<String> getPublicAdresses(List<AgentAddress> addresses){
```

Comment:
i know but i avoid to create AgentsClient object because i got some NullPointerExc and i could not handle these.